BurnOutSharp/PackerType/PECompact.cs

```diff
@@ -8,9 +11,49 @@ public string CheckContents(string file, byte[] fileContent, bool includePositio
             // "PEC2"
             byte[] check = new byte[] { 0x50, 0x45, 0x43, 0x32 };
             if (fileContent.FirstPosition(check, out int position, end: 2048))
+            {
+                string version = GetVersion(file, position);
```

Comment:
I know this isn't upheld everywhere, but can you put the checks in version order? That is, V1 first and then V2.